src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheckTest.java

```diff
@@ -723,9 +721,7 @@ public void testNoPackage() throws Exception {
         checkConfig.addAttribute("separateLineBetweenGroups", "true");
 
         createChecker(checkConfig);
-        final String[] expected = {
-            "5: " + getCheckMessage(MSG_LINE_SEPARATOR, "java.util.*"),
```

Comment:
https://github.com/checkstyle/checkstyle/blob/master/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderNoPackage.java#L5 I do not understand this behavior change.
Please explain. Your target was a empty lines between groups, not inside.

Reply:
Indeed behavior before #3551 allows single line or multiple lines inside import group, but since #3551 (PR https://github.com/checkstyle/checkstyle/pull/3677), the behavior changed to allow only single line inside import group.

I'd prefer to address this problem in https://github.com/checkstyle/checkstyle/issues/6200 with another PR. :)

Reply:
confirmed. agree.
It is small extra regression, but it is very similar to #6200 , lets fix it in there.